manifests/init.pp

```diff
@@ -977,7 +982,15 @@
   if $puppet_manage_backups == true {
     notice('Puppet will manage backups because $puppet_manage_backups is true')
     include ::gitlab::install
-    include ::gitlab::backup
```

Comment:
if that's not your intent, then please just keep the include here, as is.
heck, maybe even mark the class [private](https://github.com/puppetlabs/puppetlabs-stdlib#assert_private).